tomviz/CentralWidget.cxx

```diff
@@ -146,6 +157,7 @@ void PopulateHistogram(vtkImageData *input, vtkTable *output)
   output->AddColumn(populations.GetPointer());
 }
 
+//-----------------------------------------------------------------------------
```

Comment:
Ah, that's too bad. I find the separators visually very helpful. What editor are you using?